### ISSUE TITLE
root object cannot by an array

### ISSUE THREAD

**Issue by horsley** [josdejong/jsoneditor#663]
root object cannot by an array, get() only return the object at index 0, I am using mode 'code' so that's wierd. what I got wasn't what I typed

**Comment by josdejong** [comment]
@horsley I'm not sure what you mean. Can you post a small code example or screenshot of what you try to do?

**Comment by horsley** [comment]
I can't reproduce the problem now, I remember that I was using mode `code` and paste something like this
```
[
    {
      Name: "object1"
    },
    {
      Name: "object2"
    }
]
```

when I call api `editor.get()` I got `{Name: "object1"}` only, the second object was missing, the array structure on root node was missing too

**Comment by josdejong** [comment]
Thanks for reporting back @horsley. That sounds odd. I suggest we will close this issue for now, and please reopen as soon as you know how to reproduce.

**Comment by josdejong** [comment]
This `may` be the same issue as #687, which is fixed in now `v5.32.4`
